BEST_PRACTICES.md

```diff
@@ -123,8 +136,8 @@ Image names should be named consistently to preserve organization and developer
 
 **For example:**
 
-* `refresh-bar-button-item` / `refresh-bar-button-item@2x` and `refresh-bar-button-item-selected` / `refresh-bar-button-item-selected@2x`
```

Comment:
Is not common that people use images for borders and tinting, though. Do you have examples?